src/main/java/nl/knmi/geoweb/backend/product/sigmet/Sigmet.java

```diff
@@ -50,917 +52,991 @@
+
+			sb.append(' ').append("CNL SIGMET ").append(this.cancels).append(" ").append(validdateCancelled).append('/').append(validdateEndFormatted);
+			return sb.toString();
+		}
+		sb.append('\n');
+		
+
+		if (va_extra_fields != null) {
+			sb.append(va_extra_fields.toTAC());
+		}
+		
+		sb.append(this.phenomenon.getShortDescription());
+
+
+		sb.append('\n');
+		if (this.getObs_or_forecast()!=null) {
+			sb.append(this.obs_or_forecast.toTAC());
+			sb.append('\n');
+		}
+		sb.append(this.featureToTAC((Feature)effectiveStartGeometry, FIR));
+		sb.append('\n');
+
+		String levelInfoText=this.levelinfo.toTAC();
+		if (!levelInfoText.isEmpty()) {
+			sb.append(levelInfoText);
+			sb.append('\n');
+		}
+
+		if (this.movement_type==null) {
+			this.movement_type=SigmetMovementType.STATIONARY;
+		}
+
+		switch (this.movement_type) {
+		case STATIONARY:
+			sb.append("STNR ");
+			break;
+		case MOVEMENT:
+			if (this.movement!=null) {
+				sb.append(this.movement.toTAC());
+				sb.append('\n');
+			}
+			break;
+		case FORECAST_POSITION:
+			// Present forecast_position geometry below
+			break;
+		}
+
+		if (this.change!=null) {
+			sb.append(this.change.toTAC());
+			sb.append('\n');
+		}
+		// TODO: Check why endGeometry is not used
+		GeoJsonObject endGeometry=this.findEndGeometry(((Feature)effectiveStartGeometry).getId());
+
+		if (this.movement_type==SigmetMovementType.FORECAST_POSITION) {
+			OffsetDateTime fpaTime=this.validdate_end;
+			sb.append("FCST AT ").append(String.format("%02d", fpaTime.getHour())).append(String.format("%02d", fpaTime.getMinute())).append("Z");
+			sb.append('\n');
+			sb.append(this.featureToTAC((Feature)this.findEndGeometry(((Feature)findStartGeometry()).getId()), FIR));
+			
+		} else {
+			if (va_extra_fields !=null && va_extra_fields.no_va_expected) {
+				OffsetDateTime fpaTime=this.validdate_end;
+				sb.append("FCST AT ").append(String.format("%02d", fpaTime.getHour())).append(String.format("%02d", fpaTime.getMinute())).append("Z");
+				sb.append(" NO VA EXP");
+			} 
+		}
+		
+
+		return sb.toString();
+	}
+
+	private static String START="start";
+	private static String END="end";
+	private static String INTERSECTION="intersection";
+
+	public List<GeoJsonObject> findIntersectableGeometries() {
+		List<GeoJsonObject>objs=new ArrayList<GeoJsonObject>();
+		FeatureCollection fc=(FeatureCollection)this.geojson;
+		for (Feature f: fc.getFeatures()) {
+			if ((f.getProperty("featureFunction")!=null)&&(f.getProperty("featureFunction").equals(START)||f.getProperty("featureFunction").equals(END))){
+				objs.add(f);
+			}
+		}
+		return objs;
+	}
+
+	public List<GeoJsonObject> findEndGeometries() {
+		List<GeoJsonObject>objs=new ArrayList<GeoJsonObject>();
+		FeatureCollection fc=(FeatureCollection)this.geojson;
+		for (Feature f: fc.getFeatures()) {
+			if ((f.getProperty("featureFunction")!=null)&&f.getProperty("featureFunction").equals(END)){
+				objs.add(f);

```

Comment:
This code is left in because we might want to use it's "restrict to 69 characters length" functionality in export()

Reply:
Missing 69 character length code is noted in TODO